gammapy/cube/core.py

```diff
@@ -526,6 +526,14 @@ def to_fits(self):
 
         return hdu_list
 
+
+    def to_image_list(self):
+        """ Writes SkyCube to `gammapy.image.SkyImageList` as a list of `gammapy.image.SkyMap` objects.
+        """
+        skymaps = [SkyMap(self.name, Quantity(slice_data, self.data.unit), self.wcs) for slice_data in self.data]
```

Comment:
What's the `wcs` of the resulting skymaps like here?
It should be two-dimensional.
But I think it's three-dimensional because you just copy over the WCS from the cube?

In any case, can you please split this out into a method
```
cube.sky_image(slice_pos)
```
that returns an image?
(and add a test for that method that checks that the image WCS is 2-dimensional)